src/api/personal.rs

```diff
@@ -77,6 +77,18 @@ impl<T: Transport> Personal<T> {
                 .execute("personal_signTransaction", vec![transaction, password]),
         )
     }
+
+    /// Imports a raw key and protects it with the given password.
+    /// Returns the address of created account.
+    pub fn import_raw_key(&self, private_key: &str, password: &str) -> CallFuture<Address, T::Out> {
```

Comment:
```suggestion
    pub fn import_raw_key(&self, private_key: &[u8; 32], password: &str) -> CallFuture<Address, T::Out> {
      let private_key = hex::encode(private_key);
```
If the private key is expected to be hexadecimal, let's enforce that on the API level via typesystem.

I think using `U256` and then `hex::encode` should work too.